processor/processortest/unhealthy_processor.go

```diff
@@ -14,7 +14,8 @@ import (
 	"go.opentelemetry.io/collector/processor"
 )
 
-// NewUnhealthyProcessorCreateSettings returns a new nop settings for Create* functions.
+// Deprecated: NewUnhealthyProcessorCreateSettings returns a new nop settings for Create* functions. To be removed in future.
```

Comment:
Please follow the convention here: https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#breaking-changes by adding the version.

Please include a changelog entry